utils/fuse-overlayfs/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fuse-overlayfs
+PKG_VERSION:=1.5.0
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/containers/fuse-overlayfs/archive/v$(PKG_VERSION)
+PKG_HASH:=6c81b65b71067b303aaa9871f512c2cabc23e2b793f19c6c854d01a492b5a923
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/fuse-overlayfs
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libfuse3
+  TITLE:=fuse-overlayfs
+  URL:=https://github.com/containers/fuse-overlayfs
+endef
+
+define Package/fuse-overlayfs/description
+  FUSE overlay+shiftfs implementation for rootless containers
+endef
+
+define Package/fuse-overlayfs/install
+	$(INSTALL_DIR) $(1)/usr/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/fuse-overlayfs $(1)/usr/bin/
```

Comment:
Can this be replaced with PKG_INSTALL and PKG_INSTALL_DIR?

Reply:
Fixed. Thank you!